src/providers/dropbox.js

```diff
@@ -0,0 +1,26 @@
+export default (options) => {
+  console.log('> Dropbox')
+  return {
+    id: 'dropbox',
+    name: 'Dropbox',
+    type: 'oauth',
+    version: '2.0',
+    scope: 'account_info.read',
+    params: { grant_type: 'authorization_code' },
+    accessTokenUrl: 'https://api.dropboxapi.com/oauth2/token',
+    authorizationUrl:
+      'https://www.dropbox.com/oauth2/authorize?token_access_type=offline&response_type=code',
+    profileUrl: 'https://api.dropboxapi.com/2/users/get_current_account',
+    profile: (profile) => {
+      return {
+        id: profile.account_id,
+        name: profile.name.display_name,
+        email: profile.login,
```

Comment:
Thanks for this PR! It solves my immediate issues, hopefully it can be merged 👏 But I think this should be `profile.email`, at least from me testing this out locally.

```suggestion
        email: profile.email,
```

Reply:
Oh! Thanks you for catching this. Fixed.